vnext/Microsoft.ReactNative/ReactInstanceSettings.h

```diff
@@ -41,18 +29,30 @@ struct ReactInstanceSettings : ReactInstanceSettingsT<ReactInstanceSettings> {
   hstring JavaScriptBundleFile() noexcept;
   void JavaScriptBundleFile(hstring const &value) noexcept;
 
+  //! Should the instance run in a remote environment such as within a browser
+  //! By default, this is using a browser navigated to  http://localhost:8081/debugger-ui served
+  //! by Metro/Haul. Debugging will start as soon as the react native instance is loaded.
   bool UseWebDebugger() noexcept;
   void UseWebDebugger(bool value) noexcept;
 
+  //! Should the instance trigger the hot reload logic when it first loads the instance
```

Comment:
Love that we're adding this documentation! If we add it to the idl will it show up in the generated interface that users consume? E.g. for Intellisense.